nanome_minimization/Minimization.py

```diff
@@ -52,12 +53,13 @@ def on_advanced_settings(self):
     def on_stop(self):
         self.stop_minimization()
 
-    def start_minimization(self, ff, steps, steepest):
+    async def start_minimization(self, ff, steps, steepest):
         ff = self.convert_forcefield_value(ff)
-        def workspace_received(workspace):
-            self._process.start_process(workspace, ff, steps, steepest)
-
-        self.request_workspace(workspace_received)
+        workspace = await self.request_workspace()
+        if sum(1 for _ in workspace.complexes) > 0:
+            await self._process.start_process(workspace, ff, steps, steepest)
+        else:
+            Logs.message("No complexes found. nothing to minimize.")
```

Comment:
```suggestion
        await self._process.start_process(workspace, ff, steps, steepest)
```
`start_process` seems to already have a guard for no complexes in the workspace